lib/src/command/lish.dart

```diff
@@ -91,7 +91,7 @@ class LishCommand extends PubCommand {
 
     try {
       await log.progress('Uploading', () async {
-        var newUri = server.resolve('/api/packages/versions/new');
+        var newUri = server.resolve('api/packages/versions/new');
```

Comment:
`Uri.parse('https://pub.dev/path').resolve('api/packages/versions/new')` -> `https://pub.dev/api/packages/versions/new`

Would that be expected to be `https://pub.dev/path/api/packages/versions/new`?

`Uri.parse('https://pub.dev/path/').resolve('api/packages/versions/new')` -> `https://pub.dev/path/api/packages/versions/new`

Reply:
We normalize URLs to end in slash if path is non-empty...